pilot/pkg/model/fake_store.go

```diff
@@ -71,6 +71,8 @@ func (s *FakeStore) Create(cfg config.Config) (revision string, err error) {
 
 func (*FakeStore) Update(config config.Config) (newRevision string, err error) { return "", nil }
 
+func (*FakeStore) UpdateStatus(config config.Config) (string, error) { return "", nil }
```

Comment:
I can't find anywhere this function is actually called, outside of its own implementation and tests.  Where is this function going to be used?  Is its usage going to be in a separate PR?

Reply:
yes, in VM healthchecking

Reply:
Yes, we have two designs that will use it: auto registration and vm health checking. https://docs.google.com/document/d/10TyPy-6bOs0_7JljGpc2RneDdsaR9GktjwwaaVbz6fs/edit for more info